src/roadrunner-symfony-nyholm/composer.json

```diff
@@ -12,14 +12,16 @@
     "require": {
         "php": ">=8.0.5",
         "nyholm/psr7": "^1.4",
-        "spiral/roadrunner": "^2.0",
+        "spiral/roadrunner-cli": "^2.6",
+        "spiral/roadrunner-http": "^2.2 || ^3.5.1",
```

Comment:
Do we have really a min requirement to 2.2 and 3.5.1? Or can we go with `"^2 || ^3"`. Keep the versions wide as possible makes upgrading for others mostly easier.

Reply:
@alexander-schranz 
3.x - Starting from version **3.5.0**, support for RoadRunner **2024.1** has been added. Versions below this will only support RoadRunner **2023.3.12** and earlier. I wanted to ensure support for 2024. If ensuring support for 2024 is not necessary, the version can be downgraded to 3.0. 
2.x - Version 2.x is required to support RoadRunner **2.x**. I added 2.2 because it was the latest 2.x version, released on November 30, 2022. I assumed that users have been using it for a long time. However, we can downgrade to 2.0 if necessary.

Reply:
It makes sense, changed to ^2.0 || ^3.0.